reconnecting_ftp/__init__.py

```diff
@@ -82,8 +82,13 @@ def __init__(self,
                  max_reconnects: int = 10,
                  timeout: int = 10,
                  FTP=ftplib.FTP,
-                 encoding: str = 'utf-8') -> None:
+                 encoding: str = None) -> None:
```

Comment:
No longer providing a default for encoding.  If the user doesn't provide one, an explicit value is no longer sent, allowing ftplib.FTP to supply it's own default of 'utf-8' in versions 3.9+